src/popup/sass/layout/_search.scss

```diff
@@ -25,14 +25,16 @@
     top: 50%;
     transform: translateY(-50%);
```

Comment:
You also have to make changes in the `i` rule like making padding 0, etc. Refer to my comment above.

Reply:
Done